common-libs/common/src/services/DidDocumentService/ElemDidDocument.ts

```diff
@@ -1,17 +1,17 @@
 import base64url from 'base64url'
 import { parse } from 'did-resolver'
 
-import { op, func } from './elem-lib'
```

Comment:
One more thing: you'll need to update the versions in `package.json` and `package-lock.json`; update dependencies in dependent packages; and update changelog.

Reply:
Thank you :), I always forget this, good target for automation :)